backend.py

```diff
@@ -570,6 +570,12 @@ def partitionTargetDisk(disk, existing, preserve_first_partition, create_sr_part
         storage_partition = tool.getPartition(part_nums[STORAGE])
         if not storage_partition:
             part_nums[STORAGE] = 0
+
+        if target_platform == 'sdx8900':
```

Comment:
Is it worth extracting a small method to handle updating partition numbers for various target platforms to avoid code duplication with 8d788d63c77299034b5a0788bb991d3b042ee083? (In the potential event we will have more then it would be good to update it in one place)

Reply:
Not sure if I understood the suggestion. Do you have something specific in mind?